core/domain/services/assets/CoreAssetManager.php

```diff
@@ -76,6 +76,8 @@ class CoreAssetManager extends AssetManager
 
     const JS_HANDLE_EE_COMPONENTS = 'eventespresso-components';
 
+    const JS_HANDLE_EE_EDITOR_HOC_COMPONENTS = 'eventespresso-editor-hoc-components';
```

Comment:
higher order component components ???

plz just make it 

```
JS_HANDLE_EE_EDITOR_HOCS` = 'eventespresso-editor-hocs';
```

no one is going to not understand what that means

as well... it is likely far easier to confuse:
`JS_HANDLE_EE_EDITOR_HOC_COMPONENTS` with  `JS_HANDLE_EE_EDITOR_COMPONENTS`
than it is to confuse  `JS_HANDLE_EE_EDITOR_HOCS` with  `JS_HANDLE_EE_EDITOR_COMPONENTS`

Reply:
After discussing in slack, decision is to not only modify the name here but also modify the JS_HANDLE_EE_HOC_COMPONENTS name and actually remove the EE_ namespacing from all asset constant handles.